core/templates/domain/admin/admin-backend-api.service.ts

```diff
@@ -140,6 +140,81 @@ export class AdminBackendApiService {
       });
     });
   }
+
+  viewUsersRole(
+      filterCriterion:string, role:string, username:string
+  ):Promise<ArrayBuffer> {
+    return this.http.get<ArrayBuffer>(
```

Comment:
Why the type is ArrayBuffer? I was expecting the response to be a dict

Reply:
done!